src/config.c

```diff
@@ -2639,7 +2639,7 @@ standardConfig configs[] = {
     createIntConfig("io-threads", NULL, DEBUG_CONFIG | IMMUTABLE_CONFIG, 1, 128, server.io_threads_num, 1, INTEGER_CONFIG, NULL, NULL), /* Single threaded by default */
     createIntConfig("auto-aof-rewrite-percentage", NULL, MODIFIABLE_CONFIG, 0, INT_MAX, server.aof_rewrite_perc, 100, INTEGER_CONFIG, NULL, NULL),
     createIntConfig("cluster-replica-validity-factor", "cluster-slave-validity-factor", MODIFIABLE_CONFIG, 0, INT_MAX, server.cluster_slave_validity_factor, 10, INTEGER_CONFIG, NULL, NULL), /* Slave max data age factor. */
-    createIntConfig("list-max-ziplist-size", NULL, MODIFIABLE_CONFIG, INT_MIN, INT_MAX, server.list_max_ziplist_size, -2, INTEGER_CONFIG, NULL, NULL),
+    createIntConfig("list-max-listpack-size", "list-max-ziplist-size", MODIFIABLE_CONFIG, INT_MIN, INT_MAX, server.list_max_ziplist_size, -2, INTEGER_CONFIG, NULL, NULL),
```

Comment:
need handle the dead option when config loading like `list-max-ziplist-value`

Reply:
Do you mean add `list-max-ziplist-size` to the `ifs` of `loadServerConfigFromString`?
`list-max-ziplist-size` is a deprecated, but we also need to make `list-max-ziplist-size` backward compatible.

Reply:
yes, or the old version instance cannot be upgraded when it uses the old config file.

Reply:
i don't understand what this has to do with `list-max-ziplist-value` (an old config no longer supported since they day quicklist was introduced).

this (`list-max-ziplist-size`) is just a rename (with an alias for backwards compatibility)

Reply:
oh, I didn't see you use the alias method, so nothing need change